packages/algob/src/internal/deployer.ts

```diff
@@ -603,19 +648,41 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
 
   /**
    * This function will send Algos to ASC account in "Contract Mode"
-   * @param name     - ASC filename
+   * @param fileName     - ASC filename
    * @param flags    - Deployments flags (as per SPEC)
    * @param payFlags - as per SPEC
    * @param scTmplParams: Smart contract template parameters (used only when compiling PyTEAL to TEAL)
    */
   async fundLsig (
-    name: string,
+    fileName: string,
     flags: FundASCFlags,
     payFlags: wtypes.TxParams,
     scTmplParams?: SCParams
   ): Promise<void> {
     try {
-      await this.algoOp.fundLsig(name, flags, payFlags, this.txWriter, scTmplParams);
+      await this.algoOp.fundLsig(fileName, flags, payFlags, this.txWriter, scTmplParams);
+    } catch (error) {
+      console.log(error);
+      throw error;
+    }
+  }
+
+  /**
+   * This function will send Algos to ASC account in "Contract Mode". Takes lsig name
+   * as input
+   * @param lsigName - name of the lsig (passed by user during mkContractLsig/mkDelegatedLsig)
+   * @param flags    - Deployments flags (as per SPEC)
+   * @param payFlags - as per SPEC
+   * @param scTmplParams: Smart contract template parameters (used only when compiling PyTEAL to TEAL)
```

Comment:
this function doesn't take `scTmplParams`

Reply:
right